GGRC-39 Change help button to point to Google help page

plamut opened the pull request:
The `Help` link in the upper right corner of the page should now open the Google help page<sup>1</sup> instead of the existing internal help modal. I heard that if using Google laptop, the link takes you to directly to the GRC help page (right, @Smotko?).

Adding the `please review` label because this issue has a higher priority.

If we now need to remove the (non-used?) help modal window, the `helps` DB table and other related assets, and if this is in scope of this PR, please let me know and I will update the PR. Otherwise I suggest opening a refactoring ticket, because after all removing all that is not strictly needed for the fix.

---

<sup>1</sup>The URL needs to be configured by setting the `GRC_EXTERNAL_HELP_URL` environment variable.

Smotko commented:
My only concern is that I don't think we should be listing google internal urls in the github repository. @plamut please verify with akhil that this is ok in this case. Otherwise we will need to create a config variable like we did for the ABOUT_URL link.

plamut commented:
@Smotko Will do.

zidarsk8 commented:
@plamut backend removal is out of scope for this ticket since we want to keep the amount of changes in the release branch to the minimum.

This is the ticket for handling the whole cleanup: https://gojira.corp.google.com/browse/GGRC-439

PS: you will still have to move that url to the deployment setting.

plamut commented:
@Smotko @zidarsk8 I have updated the PR, moved the URL to the settings through the `GGRC_EXTERNAL_HELP_URL` environment variable.

I had to change some of the deployment shell scripts as well, in order to get rid of the build errors, but some of the changes might be redundant - please have a look at that (there are quite of lot of these files lying around, and it's not clear what does and what doesn't need to be changed).

I also noticed that the `#set_GGRC_EXTERNAL_HELP_URL_env_var` hint in `deploy_settings_local.sh` has no effect if the env variable is not set, which might lead other developers / QAs to think that we have a bug, since the link to the external help pages will be incorrect. Any way to improve on that?

Smotko commented:
`#set_GGRC_EXTERNAL_HELP_URL_env_var` in `deploy_settings_local.sh` does get written to app.yaml file which is where launch_gae_ggrc gets all the env variables from so it's working as expected for this case.

If you want to also make it work for `launch_ggrc` you either make sure the env var is set when the env starts up (add an export to bin/init_vagrant_env) or add the default when calling `os.environ.get("GGRC_EXTERNAL_HELP_URL")`.

plamut commented:
@Smotko Thanks. I will definitely make the improvement to avoid any confusion and false bug reports.
